pkgs/applications/version-management/git-machete/default.nix

```diff
@@ -8,17 +8,18 @@
 , nix-update-script
 , testers
 , git-machete
+, zsh
```

Comment:
`zsh` required despite completion tests not running? or is it needed for `installShellCompletion --zsh`?

Reply:
Ah, that's a left-over from my trying to get it working. I've removed it. Thanks!